js/store.js

```diff
@@ -349,21 +354,20 @@ const store = {
     return true;
   },
 
-  addFirstPartyLink(firstPartyHostname, thirdPartyHostname) {
+  async addFirstPartyLink(firstPartyHostname, thirdPartyHostname) {
     const firstParty = this.getWebsite(firstPartyHostname);
     if (!('thirdPartyHostnames' in firstParty)) {
       firstParty['thirdPartyHostnames'] = [];
     }
     if (!this.isFirstPartyLinkedToThirdParty(firstParty, thirdPartyHostname)) {
       firstParty['thirdPartyHostnames'].push(thirdPartyHostname);
-      this.setFirstParty(firstPartyHostname, firstParty);
+      await this.setFirstParty(firstPartyHostname, firstParty);
```

Comment:
I meant for the other use of ` this.addFirstPartyLink(firstPartyHostname, target);` sorry.